public/app/features/alerting/unified/components/rule-viewer/RuleViewer.tsx

```diff
@@ -73,7 +73,9 @@ const RuleViewer = () => {
 
   const isFederatedRule = isFederatedRuleGroup(rule.group);
   const isProvisioned = rulerRuleType.grafana.rule(rulerRule) && Boolean(rulerRule.grafana_alert.provenance);
-  const isPaused = rulerRuleType.grafana.alertingRule(rulerRule) && isPausedRule(rulerRule);
+  const isPaused =
+    (rulerRuleType.grafana.alertingRule(rulerRule) || rulerRuleType.grafana.recordingRule(rulerRule)) &&
```

Comment:
nit: given the logic is used in several places I think we could extract to a separate function so we can re-use it.

Reply:
There should be one `rulerRule.grafana.rule()`